.env.example

```
@@ -33,8 +33,8 @@ PUBLIC_URL=http://localhost:3333
 FILE_URL_EXPIRATION=15m
 
 # Initial Data Setup (runs only once on first startup)
-DEFAULT_TENANT_NAME=Test Organization
-DEFAULT_TENANT_SLUG=test-org
+DEFAULT_TENANT_NAME=Default Organization
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)